spec/predicates/vsa.md

```diff
@@ -1,11 +1,279 @@
-# Predicate type: SLSA Verification Summary (VSA)
+# Predicate type: Verification Summary Attestation (VSA)
 
-Type URI: https://slsa.dev/verification_summary/v1
+Type URI: https://in-toto.io/attestation/verification_summary/v1
 
-Version: 1.0
+Version: 1.1
 
 ## Purpose
 
-Records a SLSA verification decision about an artifact.
+Verification summary attestations communicate that an artifact has been verified
+against some policy and details about that verification. Such details may
+include, but are not limited to, what SLSA level the artifact and/or its
+dependencies were verified to meet.
 
-For more details, please see https://slsa.dev/spec/v1.0/verification_summary
+## Use Cases
+
+Software consumers make a decision about the validity of an
+artifact without needing to have access to all of the attestations about the
+artifact or all of its transitive dependencies.  They delegate
+complex policy decisions to some trusted party and then simply trust that
+party's decision regarding the artifact. That decision is represented by a VSA.
+
+VSAs also allow software producers to keep the details of their build pipeline
+confidential while still communicating that some verification has taken place.
+This might be necessary for legal reasons (keeping a software supplier
+confidential) or for security reasons (not revealing that an embargoed patch has
+been included).
+
+## Prerequisites
+
+Understanding of [in-toto specification] and the in-toto attestation framework.
+
+## Model
+
+A Verification Summary Attestation (VSA) is an attestation that some entity
+(`verifier`) verified one or more software artifacts (the `subject` of an
+in-toto attestation [Statement]) by evaluating the artifact and a `bundle`
+of attestations against some `policy`.  Users who trust the `verifier` may
+assume that the artifacts satisfied the indicated security polciy without
+themselves needing to evaluate the artifact or to have access to the
+attestations the `verifier` used to make its determination.
+
+The VSA also allows consumers to determine the verified levels of
+all of an artifact’s _transitive_ dependencies.  The verifier does this by
+either a) verifying the provenance of each non-source dependency listed in
+the [resolvedDependencies](/provenance/v1#resolvedDependencies) of the artifact
```

Comment:
I think this a link to SLSA Build provenance? There may be value for the generic VSA to define an internal version of a resolved dependency for dependencies without a provenance document. I'm currently thinking that along the lines that a dependency could be almost anything that is the subject of an attestation.

Reply:
Hmm, that seems like it would be pretty hard to define and reason about.

But it does seem odd for this spec to depend on SLSA build provenance.  Maybe this is an example of why VSAs might stay better with SLSA?

Reply:
I left a longer comment down below, but I'm wondering if we should just remove dependencies from VSAs. They have started to feel like a false start towards recursive SLSA, which I'd prefer to address in a dependency track or external monitoring.

Reply:
Transitive dependency verification can also be achieved by having a policy engine which performs the tree walking. It can be abstracted to be represented by the policy's URI.